pkg/manager/member/scaler.go

```diff
@@ -179,12 +181,32 @@ func ordinalPodName(memberType v1alpha1.MemberType, tcName string, ordinal int32
 // - ordinal: pod ordinal to create or delete
 // - replicas/deleteSlots: desired replicas and deleteSlots by allowing only one pod to be deleted or created
 func scaleOne(actual *apps.StatefulSet, desired *apps.StatefulSet) (scaling int, ordinal int32, replicas int32, deleteSlots sets.Int32) {
+	var ordinals []int32
```

Comment:
need to change the func's comment `allowing only one pod to be deleted or created`

Reply:
This function has the same functionality as before, why we need to change the comment ?

Reply:
maybe I misunderstand, the comment `allowing only one pod to be deleted or created` says only scale one pod.

Reply:
sorry, I misunderstand 😄